src/store/sagas/requests/adminQueryUsers.js

```diff
@@ -0,0 +1,16 @@
+import firebase from '../../../utils/Firebase'
+
+const db = firebase.firestore()
+
+export const getUnapprovedUsers = async () => {
+    let data = await db.collection('users').get()
+    let unapprovedUsers = []
+    for (let user of data.docs) {
+        let privateInfo = await db.doc(`users/${user.id}/private/info`).get()
+        if (!privateInfo.data().isApproved) {
+            let userData = await db.doc(`newUser/${privateInfo.data().email}`).get()
+             unapprovedUsers.push(userData.data())
+        }
+    }
+    return unapprovedUsers.reverse()
```

Comment:
This is not a blocker, but why do you need to reverse?

Reply:
I feel having the recent data at the top is better.

Reply:
Well, there really is no need for it. The data you're getting is arranged in alphabetical order of the `uid` ... and u know `uid` is a mayhem of alpha-numeric irrespective of who came first.

If you were to sort the data somehow (maybe by `createdAt` field), the oldest entry should be first...not the latest. But even this will need you to start creating composite indexes on firebase...so...on the long run, not needed.

but again, this is no blocker at all.

Reply:
Thanks for the heads up!